packages/pwa-buildpack/envVarDefinitions.json

```diff
@@ -0,0 +1,256 @@
+{
+    "sections": [
+        {
+            "name": "Connecting to a Magento store",
+            "variables": [
+                {
+                    "name": "MAGENTO_BACKEND_URL",
+                    "type": "url",
+                    "desc": "Connect to an instance of Magento 2.3 by specifying its public domain name.",
+                    "example": "https://magento.localhost/"
+                }
+            ]
+        },
+        {
+            "name": "Custom local origin",
+            "variables": [
+                {
+                    "name": "CUSTOM_ORIGIN_ENABLED",
+                    "type": "bool",
+                    "desc": "Get or create a secure, unique hostname/port combination and a trusted SSL certificate for local development.",
+                    "default": true
+                },
+                {
+                    "name": "CUSTOM_ORIGIN_ADD_UNIQUE_HASH",
+                    "type": "bool",
+                    "desc": "Add a unique hash string to the custom origin, based on filesystem location. This naturally separates domains when running multiple project folders on one developer machine.",
+                    "default": true
+                },
+                {
+                    "name": "CUSTOM_ORIGIN_SUBDOMAIN",
+                    "type": "str",
+                    "desc": "Specify the subdomain prefix of the custom origin manually, instead of using the package name.",
+                    "default": ""
+                },
+                {
+                    "name": "CUSTOM_ORIGIN_EXACT_DOMAIN",
+                    "type": "str",
+                    "desc": "Specify the exact domain of the custom origin manually.",
+                    "default": ""
+                }
+            ]
+        },
+        {
+            "name": "Development server",
+            "variables": [
+                {
+                    "name": "DEV_SERVER_HOST",
+                    "type": "str",
+                    "desc": "Specify the hostname the dev server should bind to. If this is set, it overrides the host chosen by custom origin settings.",
+                    "default": ""
+                },
+                {
+                    "name": "DEV_SERVER_PORT",
+                    "type": "num",
```

Comment:
"number" would read easier than "num"

Reply:
It would, but that field is used as an [`envalid` validator type](https://www.npmjs.com/package/envalid#validator-types), so we're stuck with their name scheme.

Reply:
@zetlen is there a possibility we can do a remap of these somewhere as an intermediate step? I actually had the same feedback as @jcalcaben but I did see that it was the types used by `envalid` and ended up deleting the comment.